GSA/digitalgov.gov
# HCD #6: Create Landing Page

## Summary

Creates a landing page for the HCD guides, featuring an introduction to the series and cards highlighting each guide. 

> [!WARNING]  
> Pull requests #6419, #6420, and #6424 must be merged in before this one.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-create-hcd-landing/)



### Solution
<img width="492" alt="Screenshot 2023-08-04 at 5 48 18 PM" src="https://github.com/GSA/digitalgov.gov/assets/56370370/deb6e32d-d5b2-444b-b8c1-b843d9bdffe6">

### How To Test

1.  Checkout this branch, then run `git merge jtm-create-hcd-guides-content` 
2. Navigate to `/guides/hcd`
3. Ensure that the landing page appears as intended, and that all of the links (including the one to the glossary) work. 

---

### Dev Checklist
